eng/scripts/Create-APIView.ps1

```diff
@@ -183,12 +191,12 @@ function New-SwaggerAPIViewTokens {
   $changedSwaggerFiles | ForEach-Object {
     $readmeFile = Get-SwaggerReadMeFile -swaggerFile $_
     if ($readmeFile) {
-        $swaggerReadMeFiles.Add($readmeFile) | Out-Null
+      $swaggerReadMeFiles.Add($readmeFile) | Out-Null
     }
   }
 
   LogGroupStart " Swagger APIView Tokens will be generated for the following configuration files..."
-  $readmeFile | ForEach-Object {
+  $swaggerReadMeFiles | ForEach-Object {
     LogInfo " - $_"
   }
   LogGroupEnd
```

Comment:
Below are we assuming the that default tag name doesn't change between the source and the target? What if they change the tag in the PR?

Reply:
I have updated the logic to revert to using the default tag if the new tag (source branch) is not found on the target branch